package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Final",
```

Comment:
Rename required

Reply:
Should I rename my repository?

Reply:
Does this require change in the name of the repo?
Ideally, it should have been something like actions-validation.